client/consensus/pow/src/lib.rs

```diff
@@ -534,194 +530,168 @@ pub fn import_queue<B, Transaction, Algorithm>(
 	))
 }
 
-/// Start the background mining thread for PoW. Note that because PoW mining
-/// is CPU-intensive, it is not possible to use an async future to define this.
-/// However, it's not recommended to use background threads in the rest of the
-/// codebase.
+/// Start the mining worker for PoW. This function provides the necessary helper functions that can
+/// be used to implement a miner. However, it does not do the CPU-intensive mining itself.
 ///
-/// `pre_runtime` is a parameter that allows a custom additional pre-runtime
-/// digest to be inserted for blocks being built. This can encode authorship
-/// information, or just be a graffiti. `round` is for number of rounds the
```

Comment:
This seems like useful info, why remove?

Reply:
Added it back!